src/box/sql/build.c

```diff
@@ -2921,6 +2921,7 @@ sql_id_list_append(struct IdList *list, struct Token *name_token)
 	list->a = sqlArrayAllocate(list->a, sizeof(list->a[0]), &list->nId, &i);
 	assert(i >= 0);
 	list->a[i].zName = sql_name_from_token(name_token);
+	list->a[i].has_id_lookup = name_token->z[0] != '"';
```

Comment:
Same as about `EP_Lookup` naming. And same for `has_index_lookup`.

Reply:
Removed this flag, added `char *legacy_name` field instead.